cmds/portmaster-start/lock.go

```diff
@@ -86,7 +85,7 @@ func createInstanceLock(lockFilePath string) error {
 	}
 
 	// create lock file
-	err = ioutil.WriteFile(lockFilePath, []byte(fmt.Sprintf("%d", os.Getpid())), 0666)
+	err = ioutil.WriteFile(lockFilePath, []byte(fmt.Sprintf("%d", os.Getpid())), 0666) //nolint:gosec
 	if err != nil {
```

Comment:
That permission is actually insecure. why not using something like 0600 or 0655?

Reply:
I'll add a TODO. Don't wan't add more changes to this right now.